From: =?utf-8?q?Rapha=C3=ABl_Hertzog?= <hertzog@debian.org>
Date: Wed, 15 Jul 2015 12:15:25 +0200
Subject: Drop elilo from list of d-i required packages on i386

It's gone from unstable/testing and has never been really useful on
non-ia64 systems.
---
 scripts/build/installer_debian-installer | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/build/installer_debian-installer b/scripts/build/installer_debian-installer
index f8457b1..5e78ab2 100755
--- a/scripts/build/installer_debian-installer
+++ b/scripts/build/installer_debian-installer
@@ -353,7 +353,7 @@ then
 			;;
 
 		i386)
-			DI_REQ_PACKAGES="elilo lilo grub-pc"
+			DI_REQ_PACKAGES="lilo grub-pc"
 
 			case "${LB_MODE}" in
 				ubuntu)
